src/tests/condense-number.test.ts

```diff
@@ -52,4 +52,12 @@ describe('condenseNumber()', () => {
   it('does not apply precision to Intl formatting when the locale is not supported', () => {
     expect(condenseNumber(-150000, 'IN', {maxPrecision: 2})).toBe('-150.000');
   });
+
+  it('condenses numbers when given a locale and a region', () => {
+    expect(condenseNumber(10000, 'en-CA')).toBe('10K');
+  });
+
+  it('does not condense numbers when given a malformed local and a region', () => {
```

Comment:
We already check this case above (`it('does not apply precision to Intl formatting when the locale is not supported'`)

Reply:
Should we not still test that it does so when a region is provided as well?

Reply:
I think that is covered well enough by the other tests :)